submissions/samuelJegede/database/init.sql..sql

```diff
@@ -0,0 +1,61 @@
+-- Creating foodbag database
+CREATE DATABASE food_bag;
+
+-- Selecting foodbag database
+USE food_bag;
+
+-- Creating food table
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    type VARCHAR(40) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- Changing type from VARCHAR to INT.
+ALTER TABLE food MODIFY type INT NOT NULL;
+
+-- Creating vendor_food table
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(100) NOT NULL,
+    amount FLOAT(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (admin_id) REFERENCES food (id)
+);
+
+-- Creating the admin table
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(100) NOT NULL,
+    type INT NOT NULL,
```

Comment:
Type is not just INT but a need to differentiate between 0 and 1. ENUM is better